pskb_website/static/js/editor_utils.js

```diff
@@ -248,11 +247,25 @@ function initialize_editor(local_filename, content, name, real_name, img_upload_
         }
     }, 1000));
 
+    editor.getSession().on('changeScrollTop', function(scrollTop) {
```

Comment:
Very nitpick issue here, but seems like the standard in this file is to use semicolons so makes sense to add them here too.  Would we get anything from marking this file with `use strict;`?